src/help/changelog.md

```diff
@@ -1,46 +1,26 @@
-#### August 10, 2016
+#### What's New in This Release - September 8, 2016
 
-In this version of the Broker, we separated out the validation checks into warnings and critical errors, added the interface to create D1 and D2 files, improved 508 compliance, increased server capacity, added an interface for broker users to notify other users that a submission is ready, and fixed a bug that incorrectly showed submissions as valid on the home page submission table.
+In this version of the Broker, we are importing D1 and D2 file information from the USAspending UAT environment, improved the handling of encoding in files, corrected a Resources file, and added and updated soem validations.
 
-  - [Warnings and Errors](#/help?section=warnings)
-  - [Generating D1 and D2](#/help?section=d1d2)
-  - [Improved 508 Compliance](#/help?section=508compliance)
-  - [Increased Server Capacity](#/help?section=capacity)
-  - [Notify Another User](#/help?section=notifyauser)
-  - [Bugfix: Submissions Incorrectly Show as Valid](#/help?section=homepagesubmissiontable)
+  - [Updated: Generating D1 and D2](#/help?section=d1d2updated)
+  - [Encoding in Files](#/help?section=encoding)
   - [Resources](#/help?section=resources)
   - [Updated Validations](#/help?section=updatedValidations)
   - [Browser Requirements & Known Issues](#/help?section=browser)
   - [Accessibility Statement](#/help?section=accessibilityStatement)
 
-#### Warnings and Errors{section=warnings}
+#### Updated: Generating D1 and D2{section=d1d2updated}
 
-In previous versions of the Broker, all failing validations were treated as critical errors. In this release, we have added an almost identical interface for the warnings, complete with a downloadable file and table. The warnings appear alongside the existing error download and tables. Files with warnings may be submitted, but files with critical errors will not pass validation and may not be submitted.
+In this release, we have added the ability to generate the D1 and D2 files from the data in the USAspending UAT environment that comes from ASP and FPDS. **Note:** This functionality is newly implemented so let us know if you have problems.
```

Comment:
We should add some language here to let agencies know that they need to submit data through the ASP UAT environment to create a real D2 file. Also, the procurement data only goes through 7/31/16 right now.